packages/ui/src/bounty/queries/bounty.graphql

```diff
@@ -160,3 +160,9 @@ query GetBountyContributors(
     ...BountyContributionFields
   }
 }
+
+query GetContributorWithdrawal($id: ID!) {
+  bountyFundingWithdrawalEventByUniqueInput(where: { id: $id }) {
+    id
+  }
+}
```

Comment:
Sorry about the confusion when I suggested to check the withdrawal event I just meant to add:
```gql
withdrawnInEvent { __typename }
```
in the existing:  [`fragment BountyContributionFields`](https://github.com/Joystream/pioneer/blob/78d6521e4550c1b1813e6004f2af6a459d78d075/packages/ui/src/bounty/queries/bounty.graphql#L82)

This way you can add a `isWithdrawy: !!field.asContributor` in [@/bounty/types/casts](https://github.com/Joystream/pioneer/blob/78d6521e4550c1b1813e6004f2af6a459d78d075/packages/ui/src/bounty/types/casts.ts#L83) and there's no need for an extra query, and adding an extra hook to the codebase.